frontend/src/components/uam-groups-create/tables/group-creation-users-table.tsx

```diff
@@ -0,0 +1,38 @@
+import {
```

Comment:
move to `frontend/src/components/uam-groups-create/components/users-table/users-table.tsx`